ext/project.h

```diff
@@ -67,6 +69,10 @@ typedef int SOCKET;
 #endif
 #endif /* OS_SOLARIS8 */
 
+#ifndef SO_BINDTODEVICE
+#define SO_BINDTODEVICE 25
+#endif
```

Comment:
I did some reading on this, and it's a linux-only option. I don't think we should define the value if it is not present; that's an indication that the platform does not support it.